quiche/src/lib.rs

```diff
@@ -3974,6 +3974,17 @@ impl Connection {
                 data: None,
             };
 
+            let send_at_time = match self.paths.get(send_pid) {
+                Ok(v) => {
+                    let send_time = v.recovery.get_packet_send_time();
```

Comment:
calling `get_packet_send_time()` here will give a previous packet sent time.
https://github.com/cloudflare/quiche/pull/1330/files#diff-2ac678416f8d7ea462a714681b63acedcbbb7f2c99e537ce61b7667b3bf4ac41L4032 `recovery::on_packet_sent()` will update packet's `time_sent` field with the time to sent.